README.md

```diff
@@ -38,9 +38,17 @@ module.exports = function(eleventyConfig) {
 }
 ```
 
-This will create two files: `/outbox`, which contains the root outbox declaration, and `/outbox_1`, which contains all of the actual collection items in descending date - this is for the purposes of simulating a paged Outbox object. It does this by creating, and then quietly deleting, two template files at `<source dir>/outbox/outbox.njk` and `<source dir>/outbox/outbox_page.njk`.
```

Comment:
> and `/outbox_1`

This was a bit misleading, because the generated file is `<build-dir>/outbox_page`, but it has an _id_ of `/outbox_1` (I don't understand why).

Reply:
Ah yeah, originally I was going to paginate them, but there was an issue generating the paginated files because we're putting a file at `/outbox`. The ID can be updated to `/output_page`